docs/rfc/0002-git-commit-messages.md

```diff
@@ -0,0 +1,110 @@
+# 0. Request for comments
+
+<!-- 
+    Date representing when the RFC was submittet for review 
+
+    Example:
+    Date: 08.02.2021
+-->
+Date: 12.02.2021
+
+## Status
+<!--
+    Uncomment relevant status
+-->
+
+Pending
+
+<!--
+Accepted
+Rejected
+-->
+
+## Context
+
+<!--
+    Short description of why this decision is needed
+
+    Example:
+    We want to better document our decisions
+-->
+
+We want better traceability for our git commits. We want this by prefixing commit messages by a github issue number.
+
+Without an issue number, anyone reading the code can see *what* the code does, not but not the *why*. Knowing the *why*
+is crucial when reading code you don't understand - for understanding the code in itself, to decide if it can be
+removed, make sure you can refactor it properly, and I'm sure other reasons. Especially as time goes by and the know-how
+behind the code is forgotten, this becomes important.
+
+If we follow this system, you can quickly get to the issue or PR of any code line doing git blame (or *Annotate* in
+IntelliJ).
+
+## Decision
+
+<!--
+    Short summary of the decision
+
+    Example:
+    We've decided to go for AlphaGov's system as explained [here](https://github.com/alphagov/govuk-aws/blob/24d1ea513e58ee938043d71d09815a51229067bf/docs/architecture/decisions/0001-record-architecture-decisions.md)
+-->
+
+Git commit messages found on the **master** branch should in **most cases** link to an issue, giving context to
+the commit. The issue can be a `#`followed by Github issue number or `KM-` followed by a Trello card number.
```

Comment:
Could we change the `KM` section to `KM{0..9}-`, please!

Reply:
Done, good idea!